perf/istio/gateway-bouncer/setup.sh

```diff
@@ -0,0 +1,46 @@
+#/bin/bash
+set -ex
+
+NAMESPACE=${NAMESPACE:?"New/existing NAMESPACE to install this scenario into"}
+
+function install_gateway_bouncer() {
+  local DIRNAME="${1:?"output dir"}"
+  local OUTFILE="${DIRNAME}/gateway_bouncer.yaml"
+  local INGRESS_IP=""
+
+  helm -n $NAMESPACE template \
+    --set namespace=$NAMESPACE \
+    . > "${OUTFILE}"
+
+  if [[ -z "${DRY_RUN}" ]]; then
+    kubectl -n $NAMESPACE apply -f "${OUTFILE}"
```

Comment:
Happy to see use of helm template + apply - but for this you need to use the --prune option and make sure you have a release label ( see the istio-install env.sh for an example )